control/control.SMO.xml

```diff
@@ -107,7 +107,10 @@ textdomain="control"
         <minimalistic_libzypp_config config:type="boolean">true</minimalistic_libzypp_config>
 
         <!-- boo#1124590 - Ensure correct product is selected -->
-        <select_product>SUSE-MicroOS</select_product>
+	<select_product>SUSE-MicroOS</select_product>
```

Comment:
indentation looks bbroken on github. Do you use same indentation as it was before?